hr_experience/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Experience Management",
+    "version": "12.0.1.0.0",
+    "author": "Savoir-faire Linux,"
+              "OpenSynergy Indonesia,"
+              "Numigi,"
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/oca/hr",
+    "license": "AGPL-3",
+    "category": "Human Resources",
+    "depends": ["hr"],
+    "data": [
+        "security/ir.model.access.csv",
```

Comment:
`data` sequence should be [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L29)